daemon/algod/api/algod.oas3.yml

```diff
@@ -2023,6 +2035,77 @@
         },
         "type": "object"
       },
+      "SimulationOpcodeTraceUnit": {
+        "description": "The set of trace information and effect from evaluating a single opcode.",
+        "properties": {
+          "pc": {
+            "description": "The program counter of the current opcode being evaluated.",
+            "type": "integer"
+          }
+        },
+        "required": [
+          "pc"
+        ],
+        "type": "object"
+      },
+      "SimulationPcToInnerIndex": {
+        "description": "The mapping from a program counter to an index into inner app call traces, indicating which step contract simulation branches off into an inner txn.",
+        "properties": {
+          "inner-index": {
+            "description": "The index into inner trace array, to which inner transaction get branched off from program execution.",
+            "type": "integer"
+          },
+          "pc": {
+            "description": "The program counter of the current opcode being evaluated.",
+            "type": "integer"
+          }
+        },
+        "required": [
+          "inner-index",
+          "pc"
+        ],
+        "type": "object"
+      },
+      "SimulationTransactionExecTrace": {
+        "description": "The execution trace of calling an app or a logic sig, containing the inner app call trace in a recursive way.",
+        "properties": {
+          "inner-trace": {
+            "description": "An array of SimulationTransactionExecTrace representing calling inner transactions to apps.",
```

Comment:
```suggestion
            "description": "An array of SimulationTransactionExecTrace representing the execution trace of any inner transactions executed.",
```

Reply:
see [pr response on clerk and oas2](https://github.com/algorand/go-algorand/pull/5360/commits/02cc01a00aff1061e7ff90651b5269e637309f8a)